cmd/erasure-single-drive.go

```diff
@@ -2193,7 +2193,7 @@ func (es *erasureSingle) newMultipartUpload(ctx context.Context, bucket string,
 		partsMetadata[index].Metadata = opts.UserDefined
 	}
 
-	uploadID := mustGetUUID()
+	uploadID := fmt.Sprintf("%s.%s", globalDeploymentID, mustGetUUID())
```

Comment:
```suggestion
	uploadID := fmt.Sprintf("%s.%s", globalDeploymentID, mustGetUUID())
```

You should base64 encode this for URL safety @poornas